config.yaml

```diff
@@ -38,14 +38,14 @@ languages:
 # - pageUrl: The locations of the page in different languages
 pages:
    -
-      pageId: "133436923"
+      pageId: "155236781"
       pageTitle:
          fi: "Fysiikka"
          se: "Fysik - Undervisning på finska"
          en: "Physics"
       pageUrl:
          fi: "/opetus/kurssit/fysiikka"
-         se: "/svenska/studierna/kurser/fysik_finska"
+         se: "/svenska/opetus/kurssit/fysiikka"
```

Comment:
Moi,

 

Kaikki ruotsinkieliset sivut ovat siirretty samanlaiseen kansio-hierarkiaan kuin suomenkieliset sivut:

 

/opetus/kurssit/*

/svenska/opetus/kurssit/*

 

Kun PHP-moottori lukee kopioita eikä I18N-tiedostoja käännöksille, tämä oli paras vaihtoehto pitää kaikki järjestyksessä. Nyt jos joku haluaa muuttaa suomenkielinen versio, on helpompi löytää ruotsinkielisen version.

 

Samaa ei ole vielä tehty englantikielisille sivuille.

 

~ Toffe

 

From: Samuli Thomasson [mailto:notifications@github.com] 
Sent: den 22 augusti 2015 20:38
To: SimSaladin/opetussivut <opetussivut@noreply.github.com>
Cc: Christoffer Fridlund <christoffer.fridlund@hotmail.com>
Subject: Re: [opetussivut] Documentation and bug-fixes (#1)

 

In config.yaml <https://github.com/SimSaladin/opetussivut/pull/1#discussion_r37697800> :

>        pageTitle:
>           fi: "Fysiikka"
>           se: "Fysik - Undervisning på finska"
>           en: "Physics"
>        pageUrl:
>           fi: "/opetus/kurssit/fysiikka"
> -         se: "/svenska/studierna/kurser/fysik_finska"
> +         se: "/svenska/opetus/kurssit/fysiikka"

Miksi ruotsinkieliset urlit on muutettu suomeksi, mutta englanninkieliset ovat edelleen englanniksi?

—
Reply to this email directly or view it on GitHub <https://github.com/SimSaladin/opetussivut/pull/1/files#r37697800> .